src/hypertable.c

```diff
@@ -1044,38 +1032,30 @@ ts_hypertable_get_by_id(int32 hypertable_id)
 	return ht;
 }
 
-static ChunkStoreEntry *
-hypertable_chunk_store_add(const Hypertable *h, const Chunk *chunk)
+static Chunk *
+hypertable_chunk_store_add(const Hypertable *h, const Chunk *input_chunk)
```

Comment:
Can you add a comment about what this function does? We are very bad at adding function comments and this makes it harder to understand the code.

Reply:
Done.